code-challenges.test.js

```diff
@@ -35,14 +75,44 @@ const fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
 const fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
 // Expected output: [-823, 7, 23]
 
+describe("sortIt", () => {
+    it("takes in an array and returns a new array of only odd numbers sorted from least to greatest", () =>{
+        expect(sortIt(fullArr1)).toEqual([-9, 7, 9, 199])
+        expect(sortIt(fullArr2)).toEqual([-823, 7, 23])
+    })
+})
+
+
 
 // b) Create the function that makes the test pass.
 
 
+const sortIt = (array) => {
+  return array.filter(value => typeof value === "number" && value %2 !== 0).sort((a, b) => a - b)
+}
+console.log(sortIt(fullArr1))
+console.log(sortIt(fullArr2))
```

Comment:
No need for console logging the function here since your test is invoking the function.